itou/www/stats/views.py

```diff
@@ -314,6 +314,19 @@ def render_stats_ddets(request, page_title, extra_context={}):
     return render_stats(request=request, context=context, params=params)
 
 
+@login_required
+def stats_ddets_auto_prescription(request):
+    return render_stats_ddets(request=request, page_title="Focus auto-prescription")
+
+
+@login_required
+def stats_ddets_follow_diagnosis_control(request):
+    """
+    This dashboard shows data about the diagnosis control ("Contrôle a posteriori") overall progress.
+    """
```

Comment:
gné, ce genre de commentaire sans beaucoup de valeur ajoutée me fait toujours un peu bizarre.

https://blog.codinghorror.com/code-tells-you-how-comments-tell-you-why/